PluginClasses/ZEE5PlayerSDK/PlayerManager/ZEE5PlayerManager.m

```diff
@@ -487,6 +491,40 @@ -(void)addCustomControls
     [[Zee5PlayerPlugin sharedInstance].player setRate:1.0];
 }
 
+
+-(void)CustomControlViewNew{
```

Comment:
can the method be renamed to give some more info, instead of New, it's then easier to debug in the future

Reply:
maybe name it presentParentalControlsView

Reply:
Its, Fine But in that View Adult View , ParentalView and ,Subscribve View handled

Reply:
so you mean it replaces the old CustomControlView?